tests/compile.sh

```diff
@@ -131,20 +131,30 @@ if [[ $? -eq 0 ]]; then
 fi
 set -ex
 
-if [[ "${MAKE_OPT}" == *"WW3=Y"* ]]; then
-    CMAKE_FLAGS="${CMAKE_FLAGS} -DWW3=Y"
+# Valid applications
```

Comment:
Did this option get added somewhere else? Or did how WW3 get added to the CMAKE_FLAGS change?

Reply:
@JessicaMeixner-NOAA 
`-DWW3=ON|OFF` is no longer an option a user should set.
They should define an application `S2SW` and `ATMW` will turn ON the application with waves.
The corresponding applications without WW3 are `S2S` and `ATM` without the `W`.